subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractAndroidStudioMockupCrossVersionPerformanceTest.groovy

```diff
@@ -53,13 +53,13 @@ public abstract class AbstractAndroidStudioMockupCrossVersionPerformanceTest ext
 
         void action(String className, @DelegatesTo(value = BuildActionExecuter, strategy = Closure.DELEGATE_FIRST) Closure config) {
             action {
-                tapiClassLoader.loadClass("org.gradle.tooling.BuildAction") //make sure BuildAction is available in the Gradle version we are currently running
+                AbstractAndroidStudioMockupCrossVersionPerformanceTest.this.tapiClassLoader.loadClass("org.gradle.tooling.BuildAction") //make sure BuildAction is available in the Gradle version we are currently running
```

Comment:
Did this actually make a difference?

Reply:
Forgot to explain this one. For some reason, with the new Groovy, the original does not compile as `tapiClassLoader` loses its type information in this scope:
```
gradle/subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractAndroidStudioMockupCrossVersionPerformanceTest.groovy: 62: [Static type checking] - Cannot find matching method java.lang.Object#loadClass(java.lang.String). Please check if the declared type is correct and if the method exists.
 @ line 62, column 17.
                   tapiClassLoader.loadClass(className).invokeMethod('withProjectConnection', [it, proxy] as Object[])
```